image_services/glo_30_hand/make_hand_service.py

```diff
@@ -1,10 +1,24 @@
+import argparse
 import subprocess
+import traceback
 
 import arcpy
 
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('working_directory')
+parser.add_argument(
+    '--rasters-filter',
+    default='.*Copernicus_DSM_COG_10_[NS][0-8]\d_00_[EW]\d\d\d_00_HAND.tif',
+    help=(
+        'Rasters from the glo-30-hand collection will be filtered through this '
+        'regex before they are added to the mosaic dataset.'
+    )
+)
+args = parser.parse_args()
+
 
 dataset_name = 'GLO30_HAND'
```

Comment:
There's potential for clobbering an already-existing overview during the `aws s3 cp` step, so we we may want to parameterize `dataset_name` as well.